### PR TITLE
Added fleet-server known issue 7.17.20

### PR DESCRIPTION
**Pull request by juliaElastic**
Preview: https://ingest-docs_bk_1006.docs-preview.app.elstc.co/guide/en/fleet/7.17/release-notes-7.17.20.html

### CODE REVIEW

**Comment by juliaElastic** [comment]
Build fails with this issue:
```
INFO:build_docs:All cross-document links OK
INFO:build_docs:Saving branch tracker
INFO:build_docs:Building sitemap
INFO:build_docs:Commiting changes
INFO:build_docs:Pushing changes
INFO:build_docs:Error executing: git push origin ingest-docs_bk_1006 in GIT_DIR /docs_build/.repos/target_repo.git
INFO:build_docs:---out---
INFO:build_docs:
INFO:build_docs:---err---
INFO:build_docs:Failed to add the ECDSA host key for IP address '140.82.113.3' to the list of known hosts (/tmp/.ssh/known_hosts).
INFO:build_docs:To github.com:elastic/built-docs
INFO:build_docs: ! [rejected]                    ingest-docs_bk_1006 -> ingest-docs_bk_1006 (fetch first)
INFO:build_docs:error: failed to push some refs to 'git@github.com:elastic/built-docs'
INFO:build_docs:hint: Updates were rejected because the remote contains work that you do
INFO:build_docs:hint: not have locally. This is usually caused by another repository pushing
INFO:build_docs:hint: to the same ref. You may want to first integrate the remote changes
INFO:build_docs:hint: (e.g., 'git pull ...') before pushing again.
INFO:build_docs:hint: See the 'Note about fast-forwards' in 'git push --help' for details.
INFO:build_docs:
INFO:build_docs:---------
🚨 Error: The command exited with status 1
user command error: exit status 1
```

**Review by jlind23 — APPROVED** [review]
Code change looks good to me.

**Comment by leemthompo** [comment]
@juliaElastic CI is now green after rebuild :)

File: docs/en/ingest-management/release-notes/release-notes-7.17.asciidoc [hunk 1]
```diff
@@ -59,6 +61,27 @@ Also see:
...
+[discrete]
+[[known-issues-7.17.20]]
+=== Known issues
+
+[[known-issue-3435]]
+.Fail to bootstrap Fleet Server with self-signed certificates
+[%collapsible]
+====
+*Details*
+An issue discovered in {fleet-server} that causes Fleet Server to fail to bootstrap with self-signed certificates. This is due to an update in beats dependencies where the certificate validation was completely rewritten.
```
Comment (kilfoyle) [inline docs/en/ingest-management/release-notes/release-notes-7.17.asciidoc:78]:
```suggestion
An issue has been discovered in {fleet-server} that causes it to fail to bootstrap with self-signed certificates. This is due to an update in {beats} dependencies where the certificate validation was completely rewritten.
```

**Review by kilfoyle — APPROVED** [review]
LGTM! Just two small suggestions.
Thanks for adding this @juliaElastic!

**Comment by juliaElastic** [comment]
Thanks @kilfoyle for the review! I added some more details that the issue impacts ESS too, as discussed with Julien and the team.

**Pull request merged**